### ISSUE TITLE
Epochs Vs Episodes

### ISSUE THREAD

**Issue by Akutchi** [sicara/easy-few-shot-learning#42]
Hi, 
I've been using your library for my project and I am confused on two of your notebooks.

In [1] you mention that "To train the model, we are just going to iterate over a large number of randomly generated few-shot classification tasks, and let the fit method update our model after each task. This is called episodic training."

However, in [2], you mention that you are also using episodic training but here you add the concept of epochs to it, encapsulating several episodes in each one. Is it because you now use a validation set that you did not used in [1] or that epochs in [1] are implicit because of the 40 000 episodes default value ?

I must say that I am quite confused by this. Do I have a misunderstanding somewhere?
Thank you for clarifying that for me.

[1] https://github.com/sicara/easy-few-shot-learning/blob/master/notebooks/my_first_few_shot_classifier.ipynb
[2] https://github.com/sicara/easy-few-shot-learning/blob/master/notebooks/episodic_training.ipynb

**Comment by ebennequin** [comment]
It is exactly true, in [2] I introduce validation and use the concept of epoch to define the space between two validations. So in [2], you have 200 epochs of 500 episodes each (so 100 000 episodes total), while in [1] you have 40 000 training episodes.

**Comment by Akutchi** [comment]
Thank you for your response, it it now clear for me.
